src/main/java/uk/gov/hmcts/reform/blobrouter/tasks/processors/ContainerCleaner.java

```diff
@@ -0,0 +1,59 @@
+package uk.gov.hmcts.reform.blobrouter.tasks.processors;
+
+import com.azure.storage.blob.BlobClient;
+import com.azure.storage.blob.BlobContainerClient;
+import com.azure.storage.blob.BlobServiceClient;
+import com.azure.storage.blob.models.BlobItem;
+import org.slf4j.Logger;
+import uk.gov.hmcts.reform.blobrouter.data.EnvelopeRepository;
+
+import java.util.Map;
+import java.util.UUID;
+
+import static java.util.stream.Collectors.toMap;
+import static org.slf4j.LoggerFactory.getLogger;
+import static uk.gov.hmcts.reform.blobrouter.data.model.Status.DISPATCHED;
+
+@SuppressWarnings("java:S1135") // ignore TODOs. will be removed when blob deletion part is implemented
+public class ContainerCleaner {
+
+    private static final Logger logger = getLogger(ContainerCleaner.class);
+
+    private final BlobServiceClient storageClient;
+    private final EnvelopeRepository envelopeRepository;
+
+    public ContainerCleaner(
```

Comment:
`DeleteBlobTask` would be better but current one is also fine

Reply:
This class is likely to become the task class, so best to stick to the convention Nitin mentioned.

Reply:
I assume the intention is to have a separate scheduled job class that will call this one

Reply:
then it's not a processor. it's just a service which cleans up containers 😉